apps/docs/content/guides/platform/going-into-prod.mdx

```diff
@@ -21,7 +21,8 @@ After developing your project and deciding it's Production Ready, you should run
 - Turn on [SSL Enforcement](/docs/guides/platform/ssl-enforcement)
 - Enable [Network Restrictions](/docs/guides/platform/network-restrictions) for your database.
 - Enable 2FA on GitHub. Since your GitHub account gives you administrative rights to your Supabase project, you should protect it with a strong password and 2FA using a U2F key or a TOTP app.
-- Ensure email confirmations are enabled in the `Settings > Auth` page.
+- Ensure email confirmations are [enabled](https://supabase.com/dashboard/project/_/auth/providers) in the `Settings > Auth` page.
+- Ensure that you've [set the expiry](https://supabase.com/dashboard/project/_/auth/providers) for one-time passwords (OTPs) to a reasonable value that you are comfortable with. We do not recommend setting this to a value of more than an hour.
```

Comment:
/nit since the field takes in seconds as an input
```suggestion
- Ensure that you've [set the expiry](https://supabase.com/dashboard/project/_/auth/providers) for one-time passwords (OTPs) to a reasonable value that you are comfortable with. We recommend setting this to 3600 seconds or lower
```